dockerfiles/tasks.py

```diff
@@ -146,12 +157,35 @@ def pull(c, only_latest=False):
 def test(c, arguments='', running=True):
     """Run all test suite using ``tox``."""
     if running:
-        c.run(f'{DOCKER_COMPOSE_COMMAND} exec -e GITHUB_TOKEN=$GITHUB_TOKEN web tox {arguments}', pty=True)
+        c.run(
```

Comment:
Can you please revert these changes? It seems that you have a linter/formatter that formated the whole file with a different style.

Reply:
Yes sure!!